deployments/add-ons/prometheus/README.md

```diff
@@ -0,0 +1,54 @@
+## Why you should use Prometheus with Amazon Managed Service for Prometheus (AMP)
+Many Kubeflow users utilize Prometheus and Grafana to monitor and visualize their metrics. However it can be difficult to scale open source Prometheus and Grafana as the number of nodes to be monitored increases. AMP seeks to simplify this issue by allowing multiple Prometheus servers to aggregate their metrics in Amazon Managed Prometheus, and finally Amazon Managed Grafana allows customers to then view these aggregated metrics.
+
+## Prerequisites
+First download one of our deployment options by following the directions at: https://awslabs.github.io/kubeflow-manifests/docs/deployment/prerequisites/
+
+**Note:** The steps below will assume you are sitting in the kubeflow-manifests directory.
+
+## Steps to Setup Prometheus and AMP
+1. Export cluster name and region as environment variables:
+    1. `export CLUSTER_NAME=<your-cluster-name>`
+    2. `export CLUSTER_REGION=<your-cluster-region>`
+    3. Make sure to replace the following in the above commands:
+        * **\<your-cluster-name\>**
+        * **\<your-cluster-region\>**
+2. Create an IAM Policy:
+    1. `export AMP_POLICY_NAME=<desired-amp-policy-name>`
+    2. Make sure to replace the following in the above command:
+        * **\<desired-amp-policy-name\>** - a policy name of your choosing
+    3. `export AMP_POLICY_ARN=$(aws iam create-policy --policy-name $AMP_POLICY_NAME --policy-document file://deployments/add-ons/prometheus/AMPIngestPermissionPolicy.json --query 'Policy.Arn' | tr -d '"')`
+3. Create a Service Account:
+    1. `eksctl create iamserviceaccount --name amp-iamproxy-ingest-service-account --namespace monitoring --cluster $CLUSTER_NAME --attach-policy-arn $POLICY_ARN --override-existing-serviceaccounts --approve —region $CLUSTER_REGION`
+4. Create an AMP Workspace:
+    1. `export AMP_WORKSPACE_ALIAS=<desired-amp-workspace-alias>`
+    2. Make sure to replace the following in the above command:
+        * **\<desired-workspace-alias\>** - a workspace alias of your choosing
+    3. `export AMP_WORKSPACE_ARN=$(aws amp create-workspace —alias $AMP_WORKSPACE_ALIAS —query arn | tr -d '"')`
```

Comment:
Can we create the workspace in the same region as the cluster rather than using the default region from the cli:

`export AMP_WORKSPACE_ARN=$(aws amp create-workspace —alias $AMP_WORKSPACE_ALIAS —query arn | tr -d '"')`

-->

`export AMP_WORKSPACE_ARN=$(aws amp create-workspace --region $CLUSTER_REGION —alias $AMP_WORKSPACE_ALIAS —query arn | tr -d '"')`